source/BaselineOfStargate/BaselineOfStargate.class.st

```diff
@@ -1,11 +1,10 @@
 Class {
-	#name : 'BaselineOfStargate',
-	#superclass : 'BaselineOf',
-	#category : 'BaselineOfStargate',
-	#package : 'BaselineOfStargate'
```

Comment:
There's a lot of changes due to making this commit with an old Pharo version, so please commit them again using Pharo 12 to avoid these changes.